CIScripts/Build/BuildFunctions.ps1

```diff
@@ -392,10 +381,15 @@ function Invoke-AgentTestsBuild {
         $Env:TASK_UTIL_WAIT_TIME = 10000
 
         [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
-            "", Justification="TASK_UTIL_RETRY_COUNT is used agent tests for determining " +
+            "", Justification="TASK_UTIL_RETRY_COUNT is used in agent tests for determining " +
             "timeout's threshold. They were copied from Linux unit test job.")]
         $Env:TASK_UTIL_RETRY_COUNT = 6000
 
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "", Justification="CONCURRENCY_CHECK_ENABLE is used in agent tests." +
+            " They were copied from Linux unit test job.")]
+        $Env:CONCURRENCY_CHECK_ENABLE = "true"
```

Comment:
Once `https://review.opencontrail.org/#/c/49774` is merged, this is no longer necessary to support contrail-common tests. Which tests actually need is?

Reply:
Actually only that one it seems. Gonna delete it.